src/core/trtserver.h

```diff
@@ -513,6 +516,16 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetModelRepositoryPath(
 TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetModelControlMode(
     TRTSERVER_ServerOptions* options, TRTSERVER_Model_Control_Mode mode);
 
+/// Set the model to be loaded at startup in a server options. The model will be
+/// polled from the model repositories. This function can be called multiple
```

Comment:
"will be polled from the model repositories" is confusing. Perhaps "The model must be present in one, and only one, of the specified model repositories".